<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="SaveFilters">
   <property name="geometry">
    <rect>
     <x>314</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Filters</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitMenu">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ResetToDefaults">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset to Default</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Last_Saved_Filters">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Restore Filters</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="masscutoff">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>4000</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>2400</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Mass Cutoff (neutral mass)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Peaks Included if over Mass Cutoff</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="overmasscutoff">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Peaks Included if under Mass Cutoff</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="undermasscutoff">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>2</number>
   </property>
   <property name="value">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Allowed m/z diff. between experimental and theory</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="theory_mass_diff">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <double>3.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>If two sequences are different but are within this m/z</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>210</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>and this retention time of each other, both are discarded</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="mzfilter">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="RTfilter">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Required Sequence Length</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="SeqLength">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>6</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>190</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>m/z</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>220</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>RT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Processors for Theory Generation</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="Processors">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>290</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Required number of labeling sites</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="RequiredLabels">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>330</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>99</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Minimum Abundance change for consideration</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="minabundchange">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>360</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.040000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>% labeling to use if no label present</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="zero_label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>400</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>400</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Abundance Std. Dev. Agreement Filter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Neutromer Spacing Std. Dev. Agreement Filter</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Combined Std. Dev. Agreement Filter</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="Abundstddev">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>430</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
   <property name="value">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="neutromerstddev">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>460</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
   <property name="value">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="combindedstddev">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>490</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
   <property name="value">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Outlier test ZCutoff Value</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="Zcutoff">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>520</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Error of Point 0,0 in Roll-Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>580</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Error of point with no replicates in Roll-Up</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="errorofzero">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>550</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.010000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="nonreppoint">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>580</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.050000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
